docs/configuration/plugins/configuring-slack.md

```diff
@@ -72,6 +72,18 @@ The `Slack` plugin relies on the [Events API](https://api.slack.com/events-api)
 
 > List resources command as displayed in Slack.
 
+#### `SLACK_DEPARTMENT_PROFILE_ID`
```

Comment:
nit: I think it would be easier to know what info needs to be specified if we name them like `SLACK_PROFILE_DEPARTMENT_FIELD_ID`

Reply:
👍 will update

Reply:
Updated